From: Hans-Christoph Steiner <hans@eds.org>
Subject: remove link to dead CSS on external site
Description:
 This is a pointless privacy leak since the external page in question
 does not even exist anymore.

Index: b/docs/HOWTOs/rsnapshot-HOWTO.en.html
===================================================================
--- a/docs/HOWTOs/rsnapshot-HOWTO.en.html
+++ b/docs/HOWTOs/rsnapshot-HOWTO.en.html
@@ -4,8 +4,6 @@
   <meta name="generator" content=
   "HTML Tidy for Linux/x86 (vers 12 April 2005), see www.w3.org" />
   <title>rsnapshot HOWTO</title>
-  <link href="http://www.xml-dev.com/blog/new.css" rel="stylesheet"
-  type="text/css" />
   <meta content="DocBook XSL Stylesheets V1.64.1" name=
   "generator" />
   <meta name="description" content=
